extensions/amp-bind/0.1/test/test-bind-expression.js

```diff
@@ -69,6 +69,9 @@ describe('BindExpression', () => {
   it('should evaluate ternary operator', () => {
     expect(evaluate('true ? "a" : "b"')).to.be.equal('a');
     expect(evaluate('false ? "a" : "b"')).to.be.equal('b');
+    expect(evaluate('true ?: "a"')).to.be.true;
+    expect(evaluate('"hello" ? : "a"')).to.be.equal('hello');
```

Comment:
Can we have this fail? This feels like how `a == b` is different from `a = = b`.